Mixpanel/SessionMetadata.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Metadata.swift
+//  Mixpanel
+//
+//  Created by Yarden Eitan on 10/24/17.
+//  Copyright © 2017 Mixpanel. All rights reserved.
+//
+
+import Foundation
+
+class SessionMetadata {
+    var eventsCounter: UInt64 = 0
+    var peopleCounter: UInt64 = 0
+    var sessionID: UInt64 = 0
+    var sessionStartEpoch: UInt64 = 0
+
+    func applicationDidBecomeActive() {
+        eventsCounter = 0
+        peopleCounter = 0
+        sessionID = UInt64.random
+        sessionStartEpoch = UInt64(Date().timeIntervalSince1970)
```

Comment:
i'm 99% sure this is right, but want to double check that it's in seconds and not ms

Reply:
yeah Intifying it makes it seconds as the decimal part is the milliseconds.